lib/timers.js

```diff
@@ -295,11 +295,11 @@ const unenroll = exports.unenroll = function(item) {
 // Using existing objects as timers slightly reduces object overhead.
 exports.enroll = function(item, msecs) {
   if (typeof msecs !== 'number') {
-    throw new TypeError('"msecs" argument must be a number');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'msecs', 'number');
   }
 
   if (msecs < 0 || !isFinite(msecs)) {
-    throw new RangeError('"msecs" argument must be ' +
+    throw new errors.RangeError('ERR_INVALID_ARG_TYPE', 'msecs',
                          'a non-negative finite number');
```

Comment:
hmm.. will have to think about this one because the error message is not going to format that well.